gulpfile.babel.js

```diff
@@ -71,7 +71,8 @@ parser.addArgument(["--no-jsdoc-private"], {
     help: "Jsdoc will not document private functions.",
     type: Boolean,
     action: 'storeFalse',
-    defaultValue: local_config.jsdoc_private
+    dest: 'jsdoc_private',
+    defaultValue: local_config.no_jsdoc_private
```

Comment:
The default value should be that of `jsdoc_private`. Because there is no such option as `no_jsdoc_private`. (That is, `--no-jsdoc-private` is  command line flag that turns off `jsdoc_private` but it has no value of its own.)